temporalio/worker/_worker.py

```diff
@@ -633,7 +633,7 @@ def load_default_build_id(*, memoize: bool = True) -> str:
     # * Using the loader's get_code in rare cases can cause a compile()
 
     got_temporal_code = False
-    m = hashlib.md5()
+    m = hashlib.md5(usedforsecurity=False)
```

Comment:
So it looks like this is 3.9+ only:

```suggestion
    if sys.version_info < (3, 9):
        m = hashlib.md5()
    else:
        m = hashlib.md5(usedforsecurity=False)
```